examples/netlify-edge/__integration-tests__/netlify-edge.spec.ts

```diff
@@ -0,0 +1,32 @@
+import yoga from '../src'
+import { createServer, Server } from 'http'
+import { AddressInfo } from 'net'
+import { fetch } from '@whatwg-node/fetch'
+
+describe('netlify-edge example integration', () => {
+  let server: Server
+  let port: number
+
+  beforeAll(async () => {
+    server = createServer(yoga)
+    await new Promise<void>((resolve) => server.listen(0, resolve))
+    port = (server.address() as AddressInfo).port
+  })
+
+  afterAll(async () => {
+    await new Promise((resolve) => server.close(resolve))
+  })
+
+  it('should execute query', async () => {
+    const response = await fetch(
+      `http://localhost:${port}/graphql?query=query{greetings}`,
+    )
```

Comment:
There is no need to start a server, you can just use `yoga.fetch`.

```suggestion
  it('should execute query', async () => {
    const response = await yoga.fetch(
      'http://yoga/graphql?query=query{greetings}',
    )
```